tests/utils/kagglehub.py

```diff
@@ -26,14 +26,10 @@ def do_GET(self):
         path = m.group(2)
         filepath = f"/input/tests/data/kagglehub/models/{model_handle}/{path}"
         if not os.path.isfile(filepath):
-            raise ValueError((
-                "Internet is disabled in our tests\n"
-                "kagglehub uses a fake API server\n"
-                f"Use `kagglehub.model_download('{model_handle}', path='{path}')` to download the missing file\n"
-                f"and copy it to `./docker-python/tests/data/kagglehub/models/{model_handle}/{path}`\n"
-            ))
```

Comment:
Any reason for this particular change?

Reply:
yeah, we use to throw when keras-nlp requests a file that we don't have. keras-nlp now supports optional files like "task.json". so mock needs to be updated to not throw. 

https://b.corp.google.com/issues/341982731

Reply:
Awesome thanks for the explanation!

Reply:
Can you still include that text in the error response, it will help with debugging the test.

Reply:
d.